meta-lmp-base/recipes-core/images/initramfs-ostree-lmp-image.bb

```diff
@@ -47,3 +45,10 @@ BAD_RECOMMENDATIONS += "busybox-syslog"
 
 SSTATE_SKIP_CREATION:task-image-qa = "0"
 SSTATE_SKIP_CREATION:task-image-complete = "0"
+
+python () {
+    initramfs_fstypes = d.getVar('INITRAMFS_FSTYPES')
+    image_fstypes = d.getVar('IMAGE_FSTYPES')
+    if image_fstypes != initramfs_fstypes:
+        bb.fatal('IMAGE_FSTYPES="%s" is not equal to INITRAMFS_FSTYPES="%s"' % (image_fstypes, initramfs_fstypes))
```

Comment:
Did you check if you get any fatal on the list of machines we have? Because a generic IMAGE_FSTYPES:append will affect this one, and I recall we had layers doing that.

Reply:
I have only tested it with intel machine but as you said it rise a fatal in other machines which is better than using `remove` override on the recipe.  In my opinion it is better to do it in `meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc` like we already did with machine overrides.
But this one is not related with sstate cache optimizations and I will move it to another PR.